scalalib/src/mill/javalib/android/AndroidModule.scala

```diff
@@ -706,4 +707,109 @@ trait AndroidModule extends JavaModule {
     PathRef(signedApk)
   }
 
+  /** ProGuard/R8 rules configuration files (user-provided and generated) */
+  def proguardReleaseConfigs: T[Seq[PathRef]] = {
+    Seq(
+      PathRef(moduleDir / "proguard-rules.pro"),
+      PathRef(androidModuleGeneratedSourcesFunc().androidReleaseDex.path / "proguard-rules.pro"),
+      PathRef(moduleDir / "test-proguard-rules.pro")
+    )
+  }
+
+  def proguardDebugConfigs: T[Seq[PathRef]] = {
+    Seq(
+      PathRef(moduleDir / "proguard-rules.pro"),
+      PathRef(androidModuleGeneratedSourcesFunc().androidDebugDex.path / "proguard-rules.pro"),
+      PathRef(moduleDir / "test-proguard-rules.pro")
+    )
+  }
+
+  // Additional library classes (if any) that should be treated as provided
+  def libraryClasses: T[Seq[PathRef]] = T {
```

Comment:
`libraryClasses` is a bit vague and suggest these are files rather that classpath entries, but I think it's the latter. Can you add a `Paths` or `Files` suffix to make this explicit.